Home: Improve text and readability

grahamperrin opened the pull request:
skunkworks: one word, not two. 

Whilst here, some other suggested changes.

grahamperrin committed:
index.html: suggested changes

skunkworks: one word, not two. 

Whilst here, some other suggested changes.

grahamperrin committed:
index.html: Discord instances: correcting myself

koobs commented on index.html:
```diff
@@ -39,34 +39,34 @@ <h1>BSDLabs</h1>
             </ul>
             <p>
-                Think <a href="https://en.wikipedia.org/wiki/Skunkworks_project" target="_blank">“Skunk Works”</a> for FreeBSD. Welcome to The Lab!
+                Think <a href="https://en.wikipedia.org/wiki/Skunkworks_project" target="_blank"><em>skunkworks</em></a> for FreeBSD. Welcome to the Labs!
```
* Leave Skunkworks capitalized
* Leave Lab un-pluralized. I realise the eponymous label is 'bsdlab**s**', but the 'labs' is more 'multiple projects' than 'multiple laboratories'. Maybe I'm wrong.

koobs commented on index.html:
```diff
@@ -39,34 +39,34 @@ <h1>BSDLabs</h1>
             <ul>
                 <li>Join us on <a href="https://github.com/bsdlabs" target="_blank">GitHub</a></li>
-                <li>Stop by #bsdlabs @ Libera Chat or our Discord (jump on IRC for an invite)</li>
+                <li>Stop by #bsdlabs @ Libera Chat or our Discord (for an invite: jump on IRC, or into the <a href="https://discord.com/channels/727023752348434432/757543661058654269">#community</a> channel in Discord for FreeBSD)</li>
```
* Link Discord to `https://discord.gg/9vZzzxz7y9` directly (i've done this elsewhere)
* Remove freebsd discord (#community) link, not relevent (globally, yet)
* Re-order Discord before IRC

koobs commented on index.html:
```diff
@@ -39,34 +39,34 @@ <h1>BSDLabs</h1>
-                <li><a href="https://freebsd.org" target="_blank">FreeBSD.org</a></li>
-                <li><a href="https://bugs.freebsd.org" target="_blank">FreeBSD Issue Tracker</a></li>
+                <li><a href="https://www.freebsd.org/" target="_blank">The FreeBSD Project</a></li>
+                <li><a href="https://bugs.freebsd.org/" target="_blank">FreeBSD Bugzilla</a>  (issue tracker)</li>
```
* Rename Useful to Other (or better term)
* Remove these links (www/bugs, not relevent). We'll link off to our playbook and/or other bsdlabs major repos/docs when they're public

koobs reviewed (changes requested):
Thanks for this Graham

grahamperrin committed:
index.html: respond to review

Changes requested at <https://github.com/bsdlabs/bsdlabs.github.io/pull/2#pullrequestreview-1141716920>

grahamperrin commented:
@koobs feel free to edit the file directly :-)

grahamperrin committed:
index.html: the 'someone is', 'you are' thing

The use of parentheses caused me to look twice, then a third time, at grammar. For a while, I thought I had corrected something that didn't need correcting. The bullet point reads easier without parentheses,  I reckon. 

Generally, emphasis without shouty UPPERCASE. ("THINK BIG" is exceptional, that looks fine.)

koobs commented:
> @koobs feel free to edit the file directly :-)

I don't unless asked first, some peeps like to finish up themselves. I can make post-PR edits after merge.

Thank you again for the review and updates
